### DISCUSSION TITLE
document table version column — when is it updated?

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#16545] [Q&A] [answered]
I am trying to implement a versioning system for workflow types and I noticed there's a version column in the document table.

I tried updating the workflowtype but noticed the version isn't incremented. How is this version incremented? Where and how is this version used in OC? Or is this a yessql feature?

**Comment by Piedone** [comment, marked as answer]
This is a YesSql feature, yes, see [here](https://github.com/sebastienros/yessql/blob/1ff3fd3894a6b591de8c7f9ee3301eb51ccf8972/src/YesSql.Abstractions/Document.cs#L29). You can check out the code of YesSql to see how it's used. I don't know the details, but it does get incremented when updating e.g. site settings.
